### PR TITLE
update kommander chart version

### PR DESCRIPTION
**Pull request by s12chung**

### CODE REVIEW

**Comment by jimmidyson** [comment]
We're going to update the kommander chart again in a minute when https://github.com/mesosphere/kommander-cluster-lifecycle/pull/134 is merged so you might want to wait.

**Comment by s12chung** [comment]
sure. we can wait.

**Comment by alejandroEsc** [comment]
Can you verify the chart works, I saw failures yesterday on this exact version.

File: templates/kommander.yaml [hunk 1]
```diff
@@ -30,7 +30,7 @@ spec:
   chartReference:
     chart: kommander
     repo: https://mesosphere.github.io/charts/stable
-    version: 0.1.29
+    version: 0.1.30
```
Comment (jimmidyson) [inline templates/kommander.yaml]: Just released - please update again:

```suggestion
    version: 0.1.31
```

**Pull request merged**